components/GroupSingle/GroupActions.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button } from 'ui-kit';
+import { CustomLink } from 'components';
+
+// add any additional parameters to the video urls
+const videoCallURLWithParameters = (videoURL, parameters) => {
+  const isMSIE = /*@cc_on!@*/ false || !!document.documentMode; //eslint-disable-line spaced-comment
```

Comment:
Can you explain what this is doing? Is this browser sniffing IE? Do we need to do that or can we just assume that it's Edge?

Reply:
Yes, I believe it is sniffing IE. 

@rkdavidson or @asleepingpanda do either of you know if we need to be sniffing for IE here? I pulled this over from the old codebase and can't find any good Git history on when or why this was added.

Reply:
Not sure, but it sounds like [IE doesn't handle URL/URI encoding very well](https://blog.innerht.ml/internet-explorer-has-a-url-problem/)? I can do some testing on my Windows computer this week to see if this is a global problem or what, since there's nothing unique about this video case if it's a general URL params encoding problem.